### DISCUSSION TITLE
Phoenix editor won't open

### DISCUSSION THREAD

**Discussion by SteveM2020** [phcode-dev/phoenix#1921] [Q&A] [answered]
This isn't my main computer, so when I clicked on an .md file I expected it to launch Ghostscript. Instead, it launched the Phoenix code editor. It loaded the .md file, a popup said it was updating, and it crashed and closed. I haven't been able to open it since, even after rebooting.

I never had any problem with Brackets, but this computer is using Ubuntu 24.04, and I haven't been able to get Brackets running on it, other than opening it in a virtual machine running Ubuntu 22.04. This is a pain though.....

**Comment by abose** [comment]
Sorry for the issue. Can you try to reinstall by going to phcode.io and running the install script there? That should fix the issue most likely.

**Comment by SteveM2020** [comment, marked as answer]
Thank you. This fixed it.

**Comment by Sergey43247** [comment]
![Знімок екрана 2024-11-03 185653](https://github.com/user-attachments/assets/53a78fde-0e77-46a5-9aab-b2a3b3d7238e)
невідкривається після встановлення.
